docs/README

```diff
@@ -0,0 +1 @@
+The graph is designed in this Miro board: https://miro.com/app/board/uXjVPapXnfM=/
```

Comment:
I'd suggest putting that inline in the doc (see above), so there's no confusion between `README.md` and `README` in the same directory.